aquarius/ddo_checker/schemas/v4/metadata_remote_v4.0.0.json

```diff
@@ -0,0 +1,571 @@
+        "links": {
+          "$id": "#/properties/metadata/properties/links",
+          "type": "array",
+          "default": null,
+          "items": {
+            "$id": "#/properties/metadata/properties/links/items",
+            "type": "object",
+            "default": null,
+            "additionalProperties": true,
+            "required": ["url"],
+            "properties": {
+              "name": {
+                "$id": "#/properties/metadata/properties/links/items/properties/name",
+                "type": "string",
+                "title": "The Name Schema",
+                "examples": ["example model"],
+                "pattern": "^(.*)$"
+              },
+              "type": {
+                "$id": "#/properties/metadata/properties/links/items/properties/type",
+                "type": "string",
+                "title": "sample link",
+                "default": "",
+                "examples": ["https://sample.com"],
+                "pattern": "^(.*)$"
+              },
+              "url": {
+                "$id": "#/properties/metadata/properties/links/items/properties/url",
+                "type": "string",
+                "title": "The Url Schema",
+                "default": "",
+                "examples": ["https://drive.google.com/open?id=1uuz50RGiAW8YxRcWeQVgQglZpyAebgSM"],
+                "pattern": "^(.*)$"
+              }
+            }
+          }
+        },
+        "contentLanguage": {
+          "$id": "#/properties/metadata/properties/contentLanguage",
+          "type": "string",
+          "examples": ["de-DE", "en-US"],
+          "pattern": "^(.*)$"
+        },
+        "tags": {
+          "$id": "#/properties/metadata/properties/tags",
+          "type": "array",
+          "default": null,
+          "items": {
+            "$id": "#/properties/metadata/properties/tags/items",
+            "type": "string",
+            "default": "",
+            "examples": ["image data", "classification", "animals"],
+            "pattern": "^(.*)$"
+          }
+        },
+        "categories": {
+          "$id": "#/properties/metadata/properties/categories",
+          "type": "array",
+          "default": null,
+          "items": {
+            "$id": "#/properties/metadata/properties/categories/items",
+            "type": "string",
+            "default": "",
+            "examples": ["image data", "classification", "animals"],
+            "pattern": "^(.*)$"
+          }
+        },
+        "additionalInformation": {
+          "$id": "#/properties/metadata/properties/additionalInformation",
+          "type": "object",
+          "default": null
+        },
+        "algorithm": {
+          "$id": "#/properties/metadata/properties/algorithm",
+          "type": "object",
+          "required": ["container"],
+          "properties": {
+            "language": {
+              "$id": "#/properties/metadata/properties/algorithm/properties/language",
+              "type": "string"
```

Comment:
Most other keys of `"type": "string"` include a pattern that disallows multi-line strings `"pattern": "^(.*)$"`. This seems like a sensible restriction, so for consistency, I'd like to see it applied to all keys of `"type": "string"` in this file.